qa/common/gen_qa_models.py

```diff
@@ -1076,6 +1076,103 @@ def create_plan_modelfile(
             )
 
 
+def create_plan_bf16_modelfile(
```

Comment:
Can this just be integrated into existing `create_plan_xxxfile()` functions? Any code path that is differed if dtype is bf16?